qontract_utils/tests/test_slack_api_conversations_open.py

```
@@ -0,0 +1,53 @@
+from qontract_utils.slack_api import SlackApi
+
+
+@pytest.fixture
+def mock_webclient() -> Generator[MagicMock, None, None]:
+    """Mock Slack WebClient."""
+    with patch("qontract_utils.slack_api.client.WebClient") as mock_client:
+        mock_client.return_value.retry_handlers = []
+        yield mock_client
+
+
+def test_conversations_open_returns_channel_id(mock_webclient: MagicMock) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_webclient`

(ARG001)